src/Components/CreateImageWizardV2/CreateImageWizard.tsx

```diff
@@ -136,6 +162,36 @@ const CreateImageWizard = () => {
               isSuccess={isSuccess}
             />
           </WizardStep>
+          <WizardStep
```

Comment:
I think it might be good to wrap each of these steps in their own component. For instance, we could wrap this one in `<TargetEnvironmentStep>`. 

That would make the list of steps in the wizard much easier to parse.

It will also be necessary in the future when we add the ability to customize the steps (for instance, we may or may not want to include the OpenSCAP step, etc...).

Reply:
Unfortunately, PF5 won't allow it. It checks a child of Wizard is a WizardStep. But I can try again a second time.

Reply:
Could we do something like:
```
<WizardStep>
<TargetEnvironmentStep />
</WizardStep>
```
?